packages/bits/demo/src/components/demo/wizard-v2/wizard-tooltip/wizard-tooltip-example.component.html

```diff
@@ -0,0 +1,41 @@
+<nui-wizard-horizontal #wizardComponent>
+
+    <!--  content of the 1st step -->
+    <nui-wizard-step-v2 i18n-label label="Very long header text. Very long header text">
+        <!-- content of the current step -->
+        <span i18n>Step with long header text</span>.
+
+        <!-- content of the footer -->
+        <ng-template wizardStepFooter>
+            <ng-container *ngTemplateOutlet="wizardNext"></ng-container>
+        </ng-template>
+    </nui-wizard-step-v2>
+
+    <!--  Step with custom label template -->
+    <nui-wizard-step-v2 i18n-label tooltip="Custom Label Template">
+        <ng-template wizardStepLabel>
+            <span><i>Custom Label template</i></span>
+        </ng-template>
+        <span i18n>We finally reached our last step</span>!
+        <!-- Notice how both label and footer can be placed anywhere in the content of wizard step -->
+        <ng-template wizardStepFooter>
+            <ng-container *ngTemplateOutlet="wizardPrevious"></ng-container>
+        </ng-template>
+    </nui-wizard-step-v2>
+
+    <ng-template #wizardPrevious>
+        <button type="button" nuiWizardPrevious nui-button icon="caret-left">
+            Previous
+        </button>
+    </ng-template>
+
+    <ng-template #wizardNext>
+        <button nuiWizardNext
+                type="button"
+                nui-button
+                displayStyle="primary"
+                icon="caret-right"
+                iconRight="true"
+        >Next</button>
```

Comment:
Do you mind putting the closing `>` on line 38 with the last attribute for formatting consistency?